test/basics.jl

```diff
@@ -15,9 +15,6 @@
     @test x[1] == 1
     @test length(x) == 1
 
-    @test GAP.True
-    @test ! GAP.False
```

Comment:
Why are these removed?

Reply:
Because the constants were removed, too.

They were not really needed anyway, and there is no need to even have GAPs `true` in Julia, as `true` and `false` are always converted.

The reason why they "needed" to be removed with the update was that calling GAP functions is only possible after executing the `init` function now, and the constants were defined before.